test/draw/RasterRgbTest.ooc

```diff
@@ -84,27 +84,31 @@ RasterRgbTest: class extends Fixture {
 			image := RasterRgb open(this sourceSpace)
 			image2 := image copy()
 
-			image _coordinateSystem = CoordinateSystem YUpward
+			image referenceCount decrease()
+			image = RasterRgb open(this sourceSpace, CoordinateSystem YUpward)
```

Comment:
Yeah, seems like it. We still manually edit the `image2 __coordinateSystem` since I see no way of *not* doing it. @sebastianbaginski ?